src/framework/analyses.ml

```diff
@@ -481,3 +481,34 @@ struct
   let context x = x
   (* Everything is context sensitive --- override in MCP and maybe elsewhere*)
 end
+
+(* Even more default implementations. Most transfer functions acting as identity functions. *)
+module IdentitySpec (D: Lattice.S) =
+struct
+  include DefaultSpec
+  let assign ctx (lval:lval) (rval:exp) : D.t =
+    ctx.local
+
+  let branch ctx (exp:exp) (tv:bool) : D.t =
+    ctx.local
+
+  let body ctx (f:fundec) : D.t =
+    ctx.local
+
+  let return ctx (exp:exp option) (f:fundec) : D.t =
+    ctx.local
+
+  let enter ctx (lval: lval option) (f:fundec) (args:exp list) : (D.t * D.t) list =
+    [ctx.local, ctx.local]
+
+  let combine ctx (lval:lval option) fexp (f:fundec) (args:exp list) fc (au:D.t) : D.t =
+    au
+
+  let special ctx (lval: lval option) (f:varinfo) (arglist:exp list) : D.t =
+    ctx.local
+
+  let startstate v = D.bot ()
+  let threadenter ctx lval f args = [D.top ()]
+  let threadspawn ctx lval f args fctx = ctx.local
+  let exitstate  v = D.top ()
```

Comment:
I'm not sure how correct these even are and if these same defaults could be used for any other analysis except constants. Like why is `startstate` bottom but the thread entry state top?